plugins/inputs/vsphere/endpoint.go

```diff
@@ -304,7 +304,17 @@ func (e *Endpoint) initalDiscovery(ctx context.Context) {
 func (e *Endpoint) init(ctx context.Context) error {
 	client, err := e.clientFactory.GetClient(ctx)
 	if err != nil {
-		return err
+		switch e.Parent.DisconnectedServersBehavior {
+		case "error":
+			return err
+		case "ignore":
+			// Ignore the error and postpone the init until next collection cycle
+			e.log.Warnf("Error connecting to vCenter on init: %s", err)
+			return nil
+		default:
+			return fmt.Errorf("'%s' is not a valid value for disconnected_servers_behavior",
```

Comment:
Please do not quote the string manually...
```suggestion
			return fmt.Errorf("%q is not a valid value for disconnected_servers_behavior",
```